config/Hipcheck.kdl

```diff
@@ -20,16 +20,17 @@ analyze {
     category "practices" {
         analysis "mitre/activity" policy="(lte $ 52)" weight=3
         analysis "mitre/binary" {
-			binary-file "./config/Binary.toml"
-			binary-file-threshold "0"
+			binary_file "./config/Binary.toml"
+			binary_file_threshold 0
```

Comment:
I've fixed the `binary` plugin to take kebab case in #552 , please revert this to kebab-case but leave the `0` as-is.